analyzers/tests/SonarAnalyzer.Test/TestFramework/SymbolicExecution/ValidatorTestCheck.cs

```diff
@@ -97,6 +100,9 @@ public ISymbol Symbol(string name)
         public void ValidateHasSingleExitStateAndNoException() =>
             ExitStates.Should().ContainSingle().And.ContainSingle(x => x.Exception == null);
 
+        public void ValidateSymbolConstraintsAtTag(string tag, string symbol, params SymbolicConstraint[] constraints) =>
+            TagState(tag)[Symbol(symbol)].Should().HaveOnlyConstraints(constraints);
+
```

Comment:
Instead of this, I would write an overload `TagValue(string tag, string symbolName)`.